website_event_ticket_limit/models/event_ticket.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Le Filament (https://le-filament.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class EventTicket(models.Model):
+    _inherit = "event.event.ticket"
+
+    max_ticket = fields.Integer(string="Max by order", default=0)
+
+    # ------------------------------------------------------
```

Comment:
Please remove these comment lines that are useless (and I avoid them even if there are computed fields)